nasa/cFE
# issue #425, resolve cpp check

**Describe the contribution**
Fix cppcheck warning

**Testing performed**
Steps taken to test the contribution:
1. make prep
2. make
3. make install
4. Verify still run

**System(s) tested on:**
 - Hardware
 - Ubuntu 18.04
 - CFE 6.6


**Contributor Info**
Anh Van, NASA Goddard
